### PR TITLE
Table and Panel

### PR DESCRIPTION
**Pull request by brianbodollo**
table2
table1

check sir pr po sir

### CODE REVIEW

**Comment by srph** [comment]
What's https://github.com/dev-onenetworkecommerce/june/pull/15 for?

File: docs/sections/Docs-Panels/index.jsx [hunk 1]
```diff
@@ -3,8 +3,96 @@ import React from 'react';
...
+	      	</div>
+
+	      	<h2>Panel with Tables</h2>
+	      	<div className="panel -default">
+    			<div className="panel-header">Panel Title</div>
+    			<div className="panel-body">
+    				Some default panel content here. Nulla vitae elit libero, a pharetra augue. Aenean lacinia bibendum nulla sed
+					consectetur. Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Nullam id
+					dolor id nibh ultricies vehicula ut id eli
+				</div>
```
Comment (srph) [inline docs/sections/Docs-Panels/index.jsx:52]: Indents

File: styles/components/panel.scss [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+			color: $brand-white;
+			background: $brand-crimson;
+			border-top: 1px solid $brand-crimson;
+			border-bottom-left-radius: 3px;
+			border-bottom-right-radius: 3px;
+		}
+	}
+	ul { margin-top: 0; }
+
+	.list {
```
Comment (srph) [inline styles/components/panel.scss:132]: err?

File: styles/components/panel.scss [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+		> .panel-footer {
+			padding: 10px 15px;
+			color: $brand-white;
+			background: $brand-crimson;
+			border-top: 1px solid $brand-crimson;
+			border-bottom-left-radius: 3px;
+			border-bottom-right-radius: 3px;
+		}
+	}
+	ul { margin-top: 0; }
```
Comment (srph) [inline styles/components/panel.scss]: For?

File: styles/components/panel.scss [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+		> .panel-header {
+			padding: 10px 15px;
+			color: $brand-white;
+			background: $brand-crimson;
+			border-bottom: 1px solid $brand-crimson;
+			border-top-left-radius: 3px;
+			border-top-right-radius: 3px;
+		}
+
+		> .panel-body {
```
Comment (srph) [inline styles/components/panel.scss:118]: Multiple declarations of `panel-body` with the same props and values.

**Comment by srph** [comment]
Add a screenshot, please. Also, please avoid non-descriptive PR titles.

File: docs/sections/Docs-Panels/index.jsx (hunk 1 above)
Comment (brianbodollo) [inline docs/sections/Docs-Panels/index.jsx:52]: this doesnt look like the same, my indents are oke

File: styles/components/panel.scss [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+			border-bottom-left-radius: 3px;
+			border-bottom-right-radius: 3px;
+		}
+	}
+	ul { margin-top: 0; }
+
+	.list {
+		padding-left: 0;
+		margin-bottom: 0;
+		list-style-type: none;
```
Comment (srph) [inline styles/components/panel.scss:135]: for?

File: styles/components/panel.scss (hunk 4 above)
Comment (brianbodollo) [inline styles/components/panel.scss:118]: i honestly dont have idea on what to do, thats why

**Comment by maebe22** [comment]
table2

**Comment by srph** [comment]
Again, you should put the screenshot on the first comment

File: Gemfile.lock [hunk 6]
```diff
@@ -0,0 +1,30 @@
+GEM
```
Comment (srph) [inline Gemfile.lock:1]: Like I said, remove this file, for now, at least.

File: docs/sections/Docs-Panels/index.jsx [hunk 7]
```diff
@@ -3,8 +3,96 @@ import React from 'react';
...
+								<td>Table Content</td>
+							</tr>
+						</tbody>
+					</table>
+				</div>
+    		</div>
+
+    		<h2>Panel With List Groups</h2>
+    		<div className="panel -default">
+    			<div className="panel-header">Panel Title</div>
```
Comment (srph) [inline docs/sections/Docs-Panels/index.jsx:83]: We're not using TWBS's conventions. I think you should rename these classes to:

``` sass
.panel {
  > .heading {
  }

  > .body {
  }

  > .footer {
  }
}
```

File: docs/sections/Docs-Tables/index.jsx [hunk 8]
```diff
@@ -3,8 +3,192 @@ import React from 'react';
...
+						</tr>
+
+						<tr className="-link">
+							<td>2</td>
+							<td>Table Content</td>
+							<td>Table Content</td>
+							<td>Table Content</td>
+						</tr>
+
+						<tr className="-link">
```
Comment (srph) [inline docs/sections/Docs-Tables/index.jsx:138]: `-link`?

File: styles/components/panel.scss [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+			color: $brand-white;
+			background: $brand-sun;
+			border-top: 1px solid $brand-sun;
+			border-bottom-left-radius: 3px;
+			border-bottom-right-radius: 3px;
+		}
+	}
+
+	&.-crimson {
+		border: 2px solid $brand-crimson;
```
Comment (srph) [inline styles/components/panel.scss:108]: You can just change the `border-color`.

``` sass
border-color: $brand-crimson;
```

**Comment by srph** [comment]
Why don't you make a mixin for each variant, instead of doing it manually?

``` sass
@mixin panel-variant($bg, $border, $color: $brand-white) {

}
```

File: docs/sections/Docs-Tables/index.jsx (hunk 8 above)
Comment (brianbodollo) [inline docs/sections/Docs-Tables/index.jsx:138]: just followed your style

 & > thead > tr.-link:hover,
  & > tbody > tr.-link:hover

**Comment by srph** [comment]
Why close?

**Pull request closed without merging**